### ISSUE TITLE
[DataCap Application] <Hox> - <NHXK>

### ISSUE THREAD

**Issue by Macallen35** [MikeH1999/RFfil#60]
### Data Owner Name

NanHu

### Data Owner Country/Region

China

### Data Owner Industry

IT & Technology Services

### Website

http://www.hox.net.cn/

### Social Media Handle

http://www.hox.net.cn/

### Social Media Type

Slack

### What is your role related to the dataset

Dataset Owner

### Total amount of DataCap being requested

5PiB

### Expected size of single dataset (one copy)

463TiB

### Number of replicas to store

10

### Weekly allocation of DataCap requested

512TiB

### On-chain address for first allocation

f15rbxa6g6evb4ocnjnrr4hhq6rpvo7zwwgr6eo3y

### Data Type of Application

Public, Open Commercial/Enterprise

### Custom multisig

- [ ] Use Custom Multisig

### Share a brief history of your project and organization

```text
NanHu provides full-capability products and services, integrates resources through the Internet-based telecom capability open platform, fully open and integrated, based on the cloud platform and partners to build products and services. Through years of precipitation in the communications industry, the company and its team have a strong technical team, rich communications resources, the ability to build products based on cloud platforms, and sufficient operational support capabilities.
```

### Is this project associated with other projects/ecosystem stakeholders?

No

### Describe the data being stored onto Filecoin

```text
Code base, SMS, flash message, code of AI smart robot, recording
```

### Where was the data currently stored in this dataset sourced from

My Own Storage Infra

### If you are a data preparer. What is your location (Country/Region)

China

### If you are a data preparer, how will the data be prepared? Please include tooling used and technical details?

```text
We can use lotus to prepare data. We have servers and partners.
```

### Has this dataset been stored on the Filecoin network before? If so, please explain and make the case why you would like to store this dataset again to the network. Provide details on preparation and/or SP distribution.

```text
No
```

### Please share a sample of the data

```text
https://1drv.ms/u/s!Ai4k4rlYLrp3a0GJ9w4AY_7e9Y0?e=9l493h
```

### Confirm that this is a public dataset that can be retrieved by anyone on the Network

- [X] I confirm

### What is the expected retrieval frequency for this data

Monthly

### For how long do you plan to keep this dataset stored on Filecoin

1.5 to 2 years

### In which geographies do you plan on making storage deals

Greater China, Asia other than Greater China, North America, South America, Europe

### How will you be distributing your data to storage providers

HTTP or FTP server, Shipping hard drives, Lotus built-in data transfer

### How did you find your storage providers

Slack

### Please list the provider IDs and location of the storage providers you will be working with.

```text
f03251993, Zhejiang
f03321072, Sichuan
f03241837, Shanghai
f03242023, Shanghai
f01660795, Guangdong
f03088052, Hong Kong
f03099987, Thailand
f03156722, Hongkong
f01431043, Hongkong
f02828242, Hongkong
f03099981, Beijing
f03325538, Hongkong
f01313, Hongkong
f01730296, Sai Kung
f03300709, Sichuan
f03408862, Hongkong
```

### How do you plan to make deals to your storage providers

Lotus client

### Can you confirm that you will follow the Fil+ guideline

Yes

**Comment by MikeH1999** [comment]
1. Can you confirm your identity by sending an email to [mike1999_fil@qq.com](mailto:mike1999_fil@qq.com) using your official domain name?
2. Send the supporting documents of the geographic location information of all the SPs to [mike1999_fil@qq.com](mailto:mike1999_fil@qq.com)
3. When can you expect to start sealing

**Comment by Macallen35** [comment]
@MikeH1999
ok. I sent the email to you. Please check the email from us.
We can start sealing today.
Hope you can approve us!

**Comment by MikeH1999** [comment]
<img width="460" alt="image" src="https://github.com/user-attachments/assets/0815e7aa-a0d6-4a45-ba19-7aa80fc15187" />
confirmed

**Comment by MikeH1999** [comment]
50T will be released in the first round for data validation

**Comment by Macallen35** [comment]
@MikeH1999 
Can you allocate us more datacap? Thank you.

**Comment by MikeH1999** [comment]
checker:manualTrigger

**Comment by MikeH1999** [comment]
The data is being retrieved for comparison.

**Comment by MikeH1999** [comment]
Sample confirmed
<img width="929" alt="image" src="https://github.com/user-attachments/assets/b70d21d4-b715-4084-a0f4-00568e63c489" />

**Comment by MikeH1999** [comment]
checker:manualTrigger

**Comment by MikeH1999** [comment]
checker:manualTrigger

**Comment by MikeH1999** [comment]
checker:manualTrigger

**Comment by Macallen35** [comment]
@MikeH1999 Hello allocator, we added two sps to store data.
f03099987, Bangkok  f03251993, Zhejiang

Please allocate us another round of datacap.

**Comment by MikeH1999** [comment]
checker:manualTrigger

**Comment by MikeH1999** [comment]
Can you say why retrieval rates are down overall?

**Comment by Macallen35** [comment]
@MikeH1999 Hello allocator, I have communicated with sps, and it will be recoverd these days. We add new sps here.
f03156722 HK  f02828242 HK

**Comment by MikeH1999** [comment]
checker:manualTrigger

**Comment by MikeH1999** [comment]
checker:manualTrigger

**Comment by Macallen35** [comment]
checker:manualTrigger

**Comment by MikeH1999** [comment]
checker:manualTrigger

**Comment by MikeH1999** [comment]
checker:manualTrigger

**Comment by MikeH1999** [comment]
An undisclosed sp (f03088052) with a retrieval rate of 0 appears, can you explain?

**Comment by Macallen35** [comment]
@MikeH1999 Yes, we just work with this sp and send them a little data. Their retrieval has not showed on spark dashboard, and it need some time to wait. We will  test and check if this sp can continue work with us.

**Comment by Macallen35** [comment]
@MikeH1999 Thank you for your kind reminder. I will do it as soon next time.

**Comment by MikeH1999** [comment]
SPs F03099987, F03099981, and F03156722 appear to be using a VPN.

Can you explain them?Or send another document with proof of their geographic location to [mike1999_fil@qq.com](mailto:mike1999_fil@qq.com)

**Comment by Macallen35** [comment]
@MikeH1999 We have contacted with sps.
f03156722 --- They don't know why they are considered to be using vpn. That's the ip of their server.
f03099987, f03099981 --- These two sps have updated their ip addresses. Their operation staffs made mistakes some days ago.

Thank you for your tips.

**Comment by Macallen35** [comment]
@MikeH1999 We have contacted with sps.
f03156722 --- They don't know why they are considered to be using vpn. That's the ip address of their server.
f03099987, f03099981 --- These two sps have updated their ip addresses. They used vpn because of security concerns, and now they've changed it to their servers‘ ip addresses.

Thank you for your tips.

**Comment by Macallen35** [comment]
@MikeH1999 We have sent an email to you for explanation.

**Comment by MikeH1999** [comment]
checker:manualTrigger

**Comment by MikeH1999** [comment]
Please note the rational distribution of data

**Comment by MikeH1999** [comment]
checker:manualTrigger

**Comment by Macallen35** [comment]
checker:manualTrigger

**Comment by MikeH1999** [comment]
checker:manualTrigger

**Comment by MikeH1999** [comment]
>Number of replicas to store
>10

![Image](https://github.com/user-attachments/assets/e8f4c6e4-2dad-4752-82d8-a9832052d35c)

It's up to 11 copies so far, can you explain that?

**Comment by MikeH1999** [comment]
[f03321072](https://filfox.info/en/address/f03321072)
[f03325538](https://filfox.info/en/address/f03325538)

and two undisclosed SPs

**Comment by MikeH1999** [comment]
checker:manualTrigger

**Comment by Macallen35** [comment]
@MikeH1999 Hello allocator, we go back to work just now. 
After checking, we stopped working with these two sps. f03242023 f03241837
And we changed to these two sps. f03321072, Sichuan  f03325538, Hongkong

**Comment by MikeH1999** [comment]
checker:manualTrigger

**Comment by MikeH1999** [comment]
>Number of replicas to store
>10

![Image](https://github.com/user-attachments/assets/e8f4c6e4-2dad-4752-82d8-a9832052d35c)

It's up to 11 copies so far, can you explain that?

**Comment by Macallen35** [comment]
> [@MikeH1999](https://github.com/MikeH1999) Hello allocator, we go back to work just now. After checking, we stopped working with these two sps. f03242023 f03241837 And we changed to these two sps. f03321072, Sichuan f03325538, Hongkong

@MikeH1999 Hi allocator, please look at this reply. Because we changed sps yet, that two sps got a little data from us.
And now we have total 10 sps.

**Comment by MikeH1999** [comment]
Please update the SP list

**Comment by MikeH1999** [comment]
checker:manualTrigger

**Comment by MikeH1999** [comment]
Attention to improving retrieval rates for some SPs

**Comment by MikeH1999** [comment]
checker:manualTrigger

**Comment by MikeH1999** [comment]
checker:manualTrigger

**Comment by Macallen35** [comment]
@MikeH1999 Hi allocator, we have updated our sp list. Please give us another round of datacap.
Some sps were affected by a previous spark bug that resulted in lower retrieval rates in the reports. They are repairing and waiting for the retrieval restoring.

**Comment by MikeH1999** [comment]
checker:manualTrigger

**Comment by MikeH1999** [comment]
<img width="1070" alt="Image" src="https://github.com/user-attachments/assets/61036b42-878b-4df2-bd8f-feaf9cbf7da3" />
The number of copies has grown to 12 with a large percentage of sealed copies, can you explain in detail the reason for this problem?

**Comment by MikeH1999** [comment]
checker:manualTrigger

**Comment by MikeH1999** [comment]
checker:manualTrigger

**Comment by MikeH1999** [comment]
No more DCs triggered due to multiple rule violations
